GMS/FullGMS.cpp

```diff
@@ -0,0 +1,194 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2022 (c) mindonwarp
```

Comment:
Please use full name

Reply:
Thanks for pointing this out. I was unsure about what to write in here. I will correct this and every other file header I touched.